palvelutarjotin/tests/test_oidc.py

```diff
@@ -1,13 +1,31 @@
+import contextlib
 import datetime
 import jwt
 from authlib.jose.rfc7519.claims import JWTClaims
-from django.test import TestCase
+from django.test import override_settings, TestCase
 from django.test.client import RequestFactory
+from django.utils import timezone
+from freezegun import freeze_time
 from unittest import mock
 
+from organisations.factories import UserFactory
 from palvelutarjotin.oidc import GraphQLApiTokenAuthentication
 
 
+@contextlib.contextmanager
+def mocked_jwt_request(encoded_jwt):
+    with mock.patch(
+        "helusers.oidc.ApiTokenAuthentication.decode_jwt",
+        return_value=JWTClaims(
+            jwt.decode(encoded_jwt, options={"verify_signature": False}), None
+        ),
+    ):
+        yield RequestFactory().get(
+            "/fake-path",
+            HTTP_AUTHORIZATION="Bearer {}".format(encoded_jwt),
```

Comment:
Using format string would make this more concise `f"Bearer {encoded_jwt}"`, but it's fine to leave it this way too.

Reply:
Changed